### ISSUE TITLE
CHD Support missing - Machine Disk Format

### ISSUE THREAD

**Issue by MetalSlug** [unexpectedpanda/retool#299]
At the moment, CHD dats are not fully supported.
Titles must have at least a size and one hash.
In CHD Sets there is only the disk name and the chd sha1, which represents the sha1 of the decompressed data and not the file sha1.
The file size is unknown because there can be many different versions and the data sha1 is crucial to recognize all of them.

short example:

```
<machine name="Marvel vs. Capcom 2 (USA)">
    <description>Marvel vs. Capcom 2 (USA)</description>
    <disk name="Marvel vs. Capcom 2 (USA).chd" sha1="dfdb823e6e967077ae02d104ff9608925124f691" />
  </machine>
```
  
  You can find some full dat samples at my project site:
  https://github.com/MetalSlug/MAMERedump/tree/main/MAME%20Redump

**Comment by unexpectedpanda** [comment]
Intentionally, as Redump official DATs don't use CHD. However it should be easy enough to not require a file size. I'll look into it.

**Comment by unexpectedpanda** [comment]
This will be bundled in the next major release (2.03.0). There's no ETA, as there are a few big ticket items I need to complete before I can get it out -- but it is coming.

**Comment by unexpectedpanda** [comment]
I brought this forward, and it's now available in v2.02.1.
